### ISSUE TITLE
Write end-to-end tests

### ISSUE THREAD

**Issue by mittalyashu** [serpcompany/serp-extension#30]
Run end-to-end tests on the Chrome extension feature to avoid manual testing after adding new features to tweaking something.

**Comment by mittalyashu** [comment]
@devinschumacher What do you think?

**Comment by devinschumacher** [comment]
@mittalyashu yeah or at least we start with unit tests maybe? and build up from there. e2e tests can be quite alot to do

**Comment by mittalyashu** [comment]
> @mittalyashu yeah or at least we start with unit tests maybe? and build up from there. e2e tests can be quite alot to do

Agree with that.

We can start writing unit tests for the scripts which are being executed. 👍
